faust/stores/aerospike.py

```diff
@@ -0,0 +1,198 @@
+import typing
+from typing import Any, Dict, Iterator, Optional, Tuple, Union
+
+try:  # pragma: no cover
+    import aerospike
+except ImportError:  # pragma: no cover
+    aerospike = None  # noqa
+
+from yarl import URL
+
+from faust.stores import base
+from faust.types import TP, AppT, CollectionT
+
+if typing.TYPE_CHECKING:  # pragma: no cover
```

Comment:
I'm not a Pythonic veteran, so this may be best practice, but why do we have the `typing.TYPE_CHECKING` bool check twice for different imports?

Reply:
no this does not force applications to include aerospike as a dependency even if they dont intend to use it. The tests mock this module and hence we dont need to install aerospike as a library to run tests